hummingbot/logger/reporting_proxy_handler.py

```diff
@@ -216,27 +213,19 @@ def close(self):
     async def send_aggregated_metrics_loop(self, heartbeat_interval_min: float):
         while True:
             try:
-                order_created = [e for e in self._logged_order_events if e["event_name"]
-                                 in ("BuyOrderCreatedEvent", "SellOrderCreatedEvent")]
-                if order_created:
-                    exchanges = set(e["event_source"] for e in order_created)
-                    for exchange in exchanges:
-                        markets = set(e["trading_pair"] for e in order_created if e["event_source"] == exchange)
-                        for market in markets:
-                            created_orders = [e for e in order_created if e["event_source"] == exchange and
-                                              e["trading_pair"] == market]
-                            self.send_metric("order_count", exchange, market, len(created_orders))
                 order_filled = [e for e in self._logged_order_events if e["event_name"] == "OrderFilledEvent"]
                 if order_filled:
                     exchanges = set(e["event_source"] for e in order_filled)
                     for exchange in exchanges:
                         markets = set(e["trading_pair"] for e in order_filled if e["event_source"] == exchange)
                         for market in markets:
+                            _, quote = market.split("-")
                             filled_trades = [e for e in order_filled if e["event_source"] == exchange and
                                              e["trading_pair"] == market]
-                            traded_volume = sum(e["price"] * e["amount"] for e in filled_trades)
-                            self.send_metric("filled_quote_volume", exchange, market, traded_volume)
-                            self.send_metric("trade_count", exchange, market, len(filled_trades))
+                            traded_quote_volume = sum(e["price"] * e["amount"] for e in filled_trades)
+                            traded_usdt_value = RateOracle.global_value(quote, traded_quote_volume)
```

Comment:
We can't use global_value here in a case where users set their global to something other than USDT.
You can get a rate (on rate_async) on quote-USDT, e.g.

`traded_usdt_value = await RateOralce.rate_async(f"{quote}-USDT") * traded_quote_volume`